spatial_navigation.js

```diff
@@ -896,6 +896,10 @@
   }
 
   function onKeyUp(evt) {
+    // prevent hijacking shortcuts
```

Comment:
I would prefer not leaving the comment here as the code itself already explains a lot.